config/src/lib.rs

```diff
@@ -53,7 +53,7 @@ impl Config {
             Err(err) => {
                 match err.kind {
                     figment::error::Kind::MissingField(field) => {
-                        let field = field.replace("_", "-");
+                        let field = field.replace('_', "-");
```

Comment:
Should both parameters using single quotes here?

Reply:
Nope, since the second argument is expected to be a `&str` substring, whereas the first is generic. See https://doc.rust-lang.org/std/string/struct.String.html#method.replace

<img width="796" alt="Screenshot 2022-11-29 at 12 25 14 PM" src="https://user-images.githubusercontent.com/21288394/204640300-4021c059-4cb5-426b-bd05-09809d5747ab.png">